packages/flutter/lib/src/material/bottom_app_bar.dart

```diff
@@ -155,9 +155,6 @@ class _BottomAppBarClipper extends CustomClipper<Path> {
 
   @override
   bool shouldReclip(CustomClipper<Path> oldClipper) {
```

Comment:
you can make the argument type `_BottomAppBarClipper` here, I believe the superclass explicitly makes it covariant.

Reply:
done